plugins/modules/aws_caller_info.py

```diff
@@ -75,7 +75,7 @@ def main():
         supports_check_mode=True,
     )
     if module._name == 'aws_caller_facts':
-        module.deprecate("The 'aws_caller_facts' module has been renamed to 'aws_caller_info'", version='2.13')
+        module.deprecate("The 'aws_caller_facts' module has been renamed to 'aws_caller_info'", date='2022-06-01', collection_name='amazon.aws')
```

Comment:
2.13 should be 2021-12-01
```suggestion
        module.deprecate("The 'aws_caller_facts' module has been renamed to 'aws_caller_info'", date='2021-12-01', collection_name='amazon.aws')
```

Reply:
Not intended, thanks @Qalthos!